.env.example

```
@@ -1,13 +1,33 @@
-# Required Environment Variables
-NEXT_PUBLIC_APP_URL=http://localhost:3000
+# Deployment used by `npx convex dev`
+CONVEX_DEPLOYMENT=
+NEXT_PUBLIC_CONVEX_URL=
+NEXT_PUBLIC_CONVEX_SITE_URL=
 RUSHED_CONVEX_INTERNAL_KEY=<Generate with: node -e "console.log(require('crypto').randomBytes(32).toString('base64'))">
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)